videointelligence/annotate/text_detection.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package annotate contains speech examples.
+package annotate
+
+// [START videointelligence_text_detection]
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+
+	"context"
+
+	"github.com/golang/protobuf/ptypes"
+
+	video "cloud.google.com/go/videointelligence/apiv1"
+	videopb "google.golang.org/genproto/googleapis/cloud/videointelligence/v1"
+)
+
+func textDetection(w io.Writer, filename string) error {
+	// filename := "../resources/cat.mp4"
```

Comment:
This might look odd when embedded - maybe just `cat.mp4`?

Reply:
I'll do `resources/cat.mp4` so they can at least know the file is located in the `resources` directory.